sdk/pchain/prepare_create_chain_tx.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package pchain
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/sdk/multisig"
+	"github.com/ava-labs/avalanche-cli/sdk/wallet"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+)
+
+// PrepareCreateChainTx creates uncommitted CreateChainTx
+// keychain in wallet will be used to build the transaction
+func PrepareCreateChainTx(
```

Comment:
also dont think we need to use the same name as typescript SDK. NewCreateChain that we have works better IMO